packages/blockchain-wallet-v4-frontend/src/assets/locales/en.json

```diff
@@ -1812,10 +1867,13 @@
   "modals.wallet.welcome.sb.verifybuy": "Verify your identity and buy crypto.",
   "modals.withdraw.available_for_withdrawal": "Available for Withdrawal",
   "modals.withdraw.fee": "Withdraw Fee",
+  "modals.withdraw.lock_description": "You have {locks} pending transactions. We’ll email you when these funds become available for withdrawal. <a>Learn more.</a>",
```

Comment:
The html link in this string doesn't actually link anywhere.

Reply:
Thanks @blockdylanb 😄👍

New PR with this issue fixed.
https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/5179/files